components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local AutomaticPointsTable = Class.new(
+	function(self, frame)
+		self.frame = frame
+		self.args = Arguments.getArgs(frame)
+	end
+)
+
+function AutomaticPointsTable.run(frame)
+	local pointsTable = AutomaticPointsTable(frame)
+
+	pointsTable:extractPositionBackgroundData()
+
+	pointsTable:extractTournaments()
+
+	mw.logObject(pointsTable.pbg)
+	mw.logObject(pointsTable.tournaments)
+
+	return nil
+end
+
+function AutomaticPointsTable:extractPositionBackgroundData()

```

Comment:
Needs a little doc, hard to understand without reading the code

Reply:
Do you have any suggestions?
I can explain what pbg is, but I'm not sure if that's what you mean
```suggestion
--- parses the pbg arguments, these are the background colors of specific positions
--- Usually used to indicate where a team in a specific position will end up qualifying to
function AutomaticPointsTable:parsePositionBackgroundData()
```

Reply:
No this sounds good 👍